bin/main.dart

```diff
@@ -11,7 +11,7 @@ import 'package:pana/pana.dart';
 final _gray = '\u001b[1;30m';
 final _none = '\u001b[0m';
 
-String gray(text) => "$_gray$text$_none";
```

Comment:
At that point could just use `--json`, but it is harder to read :-)
I'm fine overriding this locally, although a bit inconvenient.